### DISCUSSION TITLE
when specifying * (asterisk) while doing select, give option to generate column list in order to allow fast sql manipulation for specific columns

### DISCUSSION THREAD

**Discussion by AlexandreHGuerra** [dbeaver/dbeaver#12983] [Q&A]
**Is your feature request related to a problem? Please describe.**
when specifying * (asterisk) while doing select, give option to generate column list in order to allow fast sql manipulation for specific columns
example

i need to format 1 column in the output for a given table with 20 columns
select * from foo

to 

select 
   c1,
   c2,
   c3,
   c3,
   ad nauseam...
   format(c20,999,99)
from foo

i could have previously done a describe table foo and 
describe table foo

  c1 int
  c2 int
  c3 int
  c4 int
  ad nauseam....

to get a list of all columns, but for the sake of agility, there should be a fast way to generate all columns from a table without requiring a second step

this is very useful for ETL purposes
 
Thanks

**Comment by LonwoLonwo** [comment]
Hello @alexhguerra 
Thanks for the suggestion.
But we have this feature already!
Just use ctrl+space right after the asterisk. You can read about it here https://github.com/dbeaver/dbeaver/wiki/SQL-Assist-and-Auto-Complete

![QR7lTMeWhj](https://user-images.githubusercontent.com/45152336/123292321-9ffe2b80-d51b-11eb-8dc5-3a41e4c0bcae.gif)

Is this that you meant?

**Comment by AlexandreHGuerra** [comment]
very very cool. it should indeed be more promoted. you may close this suggestion
